### PR TITLE
adding info for automated migration

### PR DESCRIPTION
**Pull request by emuus**

### CODE REVIEW

**Comment by emuus** [comment]
@segalaj Ok discussed with Jeff and Clement and I will update the PR with these changes

File: documentation/en/source/pages/migration/migration-spec/overview.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+A migration has the following part:
+
+* ``migration``: defines the migration.
+
+.. toctree::
+   :titlesonly:
+
+   migration
+   ../../machine-images/builders/overview
```
Comment (cinlloc) [inline documentation/en/source/pages/migration/migration-spec/overview.rst]: This is already in "Target" spec, is it normal we repeat it here?

File: documentation/en/source/pages/command-line/migrate.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	      }
+	    }
+	  }
+	}
+
+
+``delete`` sub-command
+~~~~~~~~~~~~~~~~~~~~~~
+
+Deletes an existing migration. The options are:
```
Comment (cinlloc) [inline documentation/en/source/pages/command-line/migrate.rst]: "Deletes a finished migration" should be more accurate.

File: documentation/en/source/pages/command-line/migrate.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+``list`` sub-command
+~~~~~~~~~~~~~~~~~~~~
+
+Displays all the migrations for the user.
+
+``launch`` sub-command
+~~~~~~~~~~~~~~~~~~~~~~
+
+Executes an automated migration (including scan, generate and publish) of a running system, based on a yaml or json file. The options are:
```
Comment (cinlloc) [inline documentation/en/source/pages/command-line/migrate.rst]: Just a detail, but sometimes we use "yaml" and other times "YAML", I don't know if it's normal. Idem for "json"

File: documentation/en/source/pages/machine-images/builders/overview.rst [hunk 4]
```diff
@@ -5,7 +5,7 @@
 Builders
```
Comment (cinlloc) [inline documentation/en/source/pages/machine-images/builders/overview.rst]: During the meeting with Jeff, we've talked about a new section "Builders Specification", to be consistent with "Migrations Specification" and "Templates specifications".

File: documentation/en/source/pages/migration/migration-spec/migration.rst [hunk 5]
```diff
@@ -0,0 +1,99 @@
+.. Copyright (c) 2007-2018 UShareSoft, All rights reserved
+
+.. _migration:
+
+Migration
+=========
+
+The migration section describes the name, os, source and target information required to migrate a live system.
```
Comment (cinlloc) [inline documentation/en/source/pages/migration/migration-spec/migration.rst]: Does "os" have to be uppercase?

File: documentation/en/source/pages/machine-images/builders/overview.rst (hunk 4 above)
Comment (emuus) [reply documentation/en/source/pages/machine-images/builders/overview.rst]: ok

File: documentation/en/source/pages/migration/migration-spec/migration.rst (hunk 5 above)
Comment (emuus) [reply documentation/en/source/pages/migration/migration-spec/migration.rst]: ok

File: documentation/en/source/pages/command-line/migrate.rst (hunk 3 above)
Comment (emuus) [reply documentation/en/source/pages/command-line/migrate.rst]: let's raise a jira bug to do a fix for this throughout doc

File: documentation/en/source/pages/command-line/migrate.rst (hunk 2 above)
Comment (emuus) [reply documentation/en/source/pages/command-line/migrate.rst]: ok

**Comment by lqueiroga** [comment]
Thanks for the PR !

**Pull request merged**